artiq/master/scheduler.py

```diff
@@ -485,3 +485,11 @@ def check_pause(self, rid):
                     return False
                 return r.priority_key() > run.priority_key()
         raise KeyError("RID not found")
+
+    def get_wd(self, rid):
+        """Returns the ``wd`` attribute of the run with the specified RID."""
```

Comment:
``wd`` should be spelled out if this is part of the user-facing API. Attributes of `artiq.master.scheduler.Run` are not documented.